docs/portal.properties-Reference.md

```diff
@@ -181,6 +181,31 @@ installation_map_url=https://installationmap.netlify.app/
 ```
 To set up an installation map instance, one may consult the source code for the installation map [here](https://github.com/cbioportal/installation-map).
 
+## Feature toggles
+These configuration settings are used instead of checking the `app.name` property indirectly.
+```
+# Enable in order to show all cancer studies (default is false)
+show_all_cancer_studies=false
+
+# Enable in order to show the twitter preview (default is false)
+show_twitter=false
+
+# Enable in order to show development channels (default is false)
+show_dev_channels=false
+
+# Enable in order to show the study download list (default is false)
+show_study_download_list=false
+
+# Enable in order to show the reference genome genes (default is false)
+show_reference_genome_genes=false
```

Comment:
i think this is more of a performance optimization that only works for grch37. I believe the text below explains what it does?

```suggestion
# This is an experimental performance optimization. The reference genome genes endpoint is quite slow, to get around this we have saved the full reference as JSON in the frontend code. Enabling this will use that static file over the dynamically generated response from the API. If your reference genome is identical to the public instance of cBioPortal this might help for your use case as well
show_reference_genome_genes=false
```